app/scripts/views/settings.js

```diff
@@ -150,9 +150,7 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
     },
 
     beforeDestroy: function () {
-      $('body.settings').fadeOut(this.FADE_OUT_SETTINGS_MS, function (){
-        $('body').removeClass('settings').show();
-      });
+      $('body').removeClass('settings');
```

Comment:
@vladikoff Yeah, the reason it works for ToS and PP is because of that CSS rule: it hides the other layout content until the `settings` class is removed from body.